engines/python/setup/setup.py

```diff
@@ -57,7 +57,8 @@ def run(self):
 
     test_requirements = [
         'numpy', 'requests', 'Pillow', 'transformers', 'torch', 'einops',
-        'accelerate', 'sentencepiece', 'protobuf', 'yapf', 'pydantic==1.10.13'
+        'accelerate', 'sentencepiece', 'protobuf', 'yapf', 'pydantic==1.10.13',
+        'transformers_neuronx'
```

Comment:
Is the unit test cases not checked in yet?

Reply:
oh, why we add tnx as dependencies? Could we do mocking?

Reply:
yea i guess we could mock. I wanted to avoid replicating the enum defined by transformers neuronx in the main code. It might be easiest to just replicate